engine/src/main/java/org/hibernate/validator/engine/HibernateConstraintViolation.java

```diff
@@ -0,0 +1,7 @@
+package org.hibernate.validator.engine;
+
+import javax.validation.ConstraintViolation;
+
+public interface HibernateConstraintViolation<T> extends ConstraintViolation<T> {
+	Object getInfo();
```

Comment:
Obviously its up to the user to make sure this additional context is serializable in case the _ConstraintViolation_ should be serialized. But serialization is problematic anyways due to the root bean being part of the _ConstraintViolation_